pytorch_lightning/strategies/deepspeed.py

```diff
@@ -372,10 +515,6 @@ def _set_node_environment_variables(self) -> None:
         os.environ["WORLD_SIZE"] = str(self.world_size)
         os.environ["LOCAL_RANK"] = str(self.local_rank)
 
-    @property
-    def restore_checkpoint_after_setup(self) -> bool:
```

Comment:
Why was it moved?

Reply:
Note: Moving code makes it harder for the reviewer to understand what is added and just moved. I would advise keeping the code re-organization to another PR.